test-kit/apps/html-parser/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.redhat.training</groupId>
+	<artifactId>html-parser</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>html-parser</name>
+	<description>GE: Testing Routes with Camel Test Kit</description>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<fuse.version>7.9.0.fuse-sb2-790065-redhat-00001</fuse.version>
```

Comment:
```suggestion
		<fuse.version>7.10.0.fuse-sb2-7_10_0-00014-redhat-00001</fuse.version>
```

Reply:
If you accept the suggestion, please also update the pom in the solutions folder because I can't do a suggestion.. XD